Tools/CISupport/ews-build/steps.py

```diff
@@ -7193,6 +7201,27 @@ class ScanBuildWithoutChange(ScanBuild):
     name = 'scan-build-without-change'
     output_directory = SCAN_BUILD_OUTPUT_DIR + '-baseline'
 
+    def __init__(self, analyze_results=True, **kwargs):
+        self.analyzeResults = analyze_results
+        super().__init__(**kwargs)
+
+    def addFollowUpSteps(self, rc):
+        steps_to_add = self.uploadLogsSteps()
+        # If running this step after a successful build, proceed to analyze results.
+        if self.analyzeResults:
```

Comment:
analyzeResults seems like confusing variable name, it's hard to follow why we don't analyze the SUCCESS/FAILURE status of this step if analyzeResults is True.